src/SearchResults.js

```diff
@@ -15,14 +15,13 @@ export default function Search({
   fetchSearchResults,
   showHeader = false,
 }) {
-  // eslint-disable-next-line camelcase
-  const page_size = 10;
+  const limit = 10;
```

Comment:
We renamed the variable denoting the number of results per page from `page_size` to `limit` [as amiljack suggested](https://github.com/amilajack/tide/pull/290#discussion_r546017123).

Because we do that, the comment `// eslint-disable-next-line camelcase` is no longer necessary, so we remove it.